pandas/tests/extension/test_sparse.py

```diff
@@ -112,7 +112,15 @@ def test_view(self, data):
 
 
 class TestConstructors(BaseSparseTests, base.BaseConstructorsTests):
-    pass
+    @pytest.mark.xfail(reason="GH-26469", strict=False)
+    def test_series_constructor_no_data_with_index(self, data, na_value):
+        # ValueError: Cannot convert non-finite values (NA or inf) to integer
+        super().test_series_constructor_no_data_with_index(data, na_value)
```

Comment:
Do you know why sparse is failing?

Reply:
hmm, so much for the xfail. no longer failing. was only failing on only one of the fill values before hence the strict=False.